test/serializable_resource_test.rb

```diff
@@ -2,8 +2,13 @@
 
 module ActiveModel
   class SerializableResourceTest < Minitest::Test
+    class Profile < ::Model
```

Comment:
I think a more sane approach would be to rename our `Model` class into `PoroModel`. Thoughts?